### ISSUE TITLE
iOS - Sticky toolbar does not stick

### ISSUE THREAD

**Issue by marijnh** [ProseMirror/prosemirror-menu#3]
See https://github.com/ProseMirror/prosemirror/issues/223

**Comment by Dev1an** [comment]
What about using sticky positioning in CSS instead of Javascript?

See:
 - [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/position) on `position: sticky`
 - [Browser compatibility](https://caniuse.com/#feat=css-sticky)

**Comment by marijnh** [comment]
Yes, we know about position: sticky. Back in 2016 it was terribly broken in iOS. Not sure what the current situation is.

**Comment by Dev1an** [comment]
The stickyness with CSS works on iOS. I tried it locally and it works fine. Shall I create a new pull request with this feature?

**Comment by marijnh** [comment]
Sure! Though I think we'd want to keep the existing implementation for browsers where sticky isn't supported.
